network/network.go

```diff
@@ -381,10 +393,18 @@ func (n *network) HealthCheck(context.Context) (interface{}, error) {
 		errorReasons = append(errorReasons, fmt.Sprintf("not connected to a minimum of %d peer(s) only %d", n.config.HealthConfig.MinConnectedPeers, connectedTo))
 	}
 	if !wasMsgReceivedRecently {
-		errorReasons = append(errorReasons, fmt.Sprintf("no messages from network received in %s > %s", timeSinceLastMsgReceived, n.config.HealthConfig.MaxTimeSinceMsgReceived))
+		if !msgReceived {
+			errorReasons = append(errorReasons, "no messages received from network")
+		} else {
+			errorReasons = append(errorReasons, fmt.Sprintf("no messages from network received in %s > %s", timeSinceLastMsgReceived, n.config.HealthConfig.MaxTimeSinceMsgReceived))
+		}
 	}
```

Comment:
Rather than:
```golang
if !wasMsgReceivedRecently {
	if !msgReceived {
		errorReasons = append(errorReasons, "no messages received from network")
	} else {
		errorReasons = append(errorReasons, fmt.Sprintf("no messages from network received in %s > %s", timeSinceLastMsgReceived, n.config.HealthConfig.MaxTimeSinceMsgReceived))
	}
}
```
Can we do:
```golang
if !msgReceived {
	errorReasons = append(errorReasons, "no messages received from network")
} else if !wasMsgReceivedRecently {
	errorReasons = append(errorReasons, fmt.Sprintf("no messages from network received in %s > %s", timeSinceLastMsgReceived, n.config.HealthConfig.MaxTimeSinceMsgReceived))
}
```
Same below.

Reply:
yup